### PR TITLE
feat(docker): create telos image

### PR DESCRIPTION
**Pull request by leisterfrancisco**
## What does this PR do?
- Clone Telos contract to build it
- Set eosio.cdt and eosio version according to Telos [release/1.6.x](https://github.com/telosnetwork/telos.contracts/tree/release/1.6.x) dependencies specifications
- Set staring script

Waiting for answer in [telosnetwork/telos.contracts](https://github.com/telosnetwork/telos.contracts/issues/19) on a reported issue

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,21 +7,21 @@
 # Telos Mainnet Local Network
```
Comment (leisterfrancisco) [inline README.md]: I don't feel it is a good name, what do you think @xavier506?

File: .github/workflows/publish-docker-image.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            eoscostarica506/telos-local
+            ghcr.io/${{ github.repository }}
+
+      - name: Build new images and push
+        run: |
+          make \
+            build-docker \
+            push-image
+        env:
+          TESTNET_EOSIO_PRIVATE_KEY: ${{ secrets.TESTNET_EOSIO_PRIVATE_KEY }}
```
Comment (xavier506) [inline .github/workflows/publish-docker-image.yaml]: @leisterfrancisco do we need these as repo secrets or in .env?

File: README.md (hunk 1 above)
Comment (xavier506) [reply README.md]: Telos Local Development Network?

**Review by xavier506 — CHANGES_REQUESTED** [review]
@leisterfrancisco  please see my comments

can wee add a note about the system contracts on the readme file also

File: README.md [hunk 3]
```diff
@@ -4,7 +4,10 @@
 	</a>
 </p>
 
-# Telos Mainnet Local Network
+# Telos Local Development Network
+
+## Note
```
Comment (leisterfrancisco) [inline README.md:9]: @xavier506 what do you think?

Comment (xavier506) [reply README.md:9]: perfect!

**Pull request merged**